extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -28,37 +28,82 @@ public static function onBeforePageDisplay() {
 		return true;
 	}
 
-	public static function render( $input, array $args ) {
-		global $wgCityId;
+	/**
+	 * Checks arguments for errors.
+	 * @param array $args
+	 * @oaram errorMessage Return parameter with the proper error message to show. Disregard if return is false
+	 * @return true if ok, false if error
+	 */
+	private static function checkArguments( array $args, $modelData, &$errorMessage ) {
+		// mostrecent must be bool
+		if ( array_key_exists( 'mostrecent', $args ) && $args['mostrecent'] !== 'true' && $args['mostrecent'] !== 'false' ) {
```

Comment:
Minor: `isset( $args['mostrecent'] )` is probably slightly better here and in the other checks. This condition should also be split over multiple lines